fix a channel input

taizan opened the pull request:
fill 255 to the image which has alpha channel

taizan committed:
fix a channel input

taizan committed:
Merge pull request #1 from taizan/a_ch-patch

fix a channel input

uyongw commented:
With this PR, the image with a channel is correct now. But it seems has issue on image without a channel? Or does it miss to return the grayed image for RGB?

```
diff --git a/cgi-bin/paint_x2_unet/img2imgDataset.py b/cgi-bin/paint_x2_unet/img2imgDataset.py
index cc521de..3d424a7 100755
--- a/cgi-bin/paint_x2_unet/img2imgDataset.py
+++ b/cgi-bin/paint_x2_unet/img2imgDataset.py
@@ -36,6 +36,7 @@ def cvt2GRAY(img):
         return cv2.cvtColor(img2, cv2.COLOR_RGB2GRAY)
     else:
         # RGB image
+        return cv2.cvtColor(img, cv2.COLOR_RGB2GRAY)

 class ImageAndRefDataset(chainer.dataset.DatasetMixin):
```

whughw committed:
Update img2imgDataset.py

Fix the bug in function cvt2GRAY: it couldn't handle with RGB image because of a missing line of code, which resulted in a runtime error and caused an exception.

taizan committed:
Merge pull request #2 from whughw/patch-1

Update img2imgDataset.py
